media_management_api/requirements/base.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.11.23
+Django~=2.0.0
```

Comment:
Am I correct in understanding that this will install django v2.0.13?

Reply:
Yup, it will just get the latest patch for 2.0